docs/geedocs/5.3.1/answer/3481558.html

```diff
@@ -280,6 +309,128 @@ <h3>Commands</h3>
   <pre>
 <code><b>--map</b> map project </code></pre>
   <p><i>Optional</i>. The map project to be added to the database.</p>
+
+
+  <h2>gemodifyimageryproject</h2>
+  <pre>
+ <code><b>gemodifyimageryproject</b> [--<strong>mercator</strong> | --<strong>flat]</strong> [--<strong>historical_imagery</strong> | --<strong>no_historical_imagery</strong>]
+   -o <em>projectname</em> {[--<strong>maxlevel</strong> <em>level</em> | --<strong>maxleveloverride</strong> <em>level</em>] <em>insetresource</em>}</code>...</pre>
+  
+
+  <h3>Purpose</h3>
+  <p>Modifies an existing imagery project.</p>
+  
+  <h3>Commands</h3>
+  <pre><code><b>--mercator </b></code></pre>
+  <p><i>Optional</i>. Uses Mercator map projection for the imagery project.</p>
+  <pre><code><b>--flat </b></code></pre>
+  <p><i>Default</i>. Uses Flat map (Plate Carrée) projection.</p>
+  <pre><code><b>--historical_imagery </b></code></pre>
+  <p><i>Optional</i>. Uses historical imagery for the project.</p>
+  <pre><code><b>--no_historical_imagery </b></code></pre>
+  <p><i>Default</i>. Uses normal imagery for the project.</p>
+  <h3>Options</h3>
+  <pre>
+  <code><b>--maxlevel </b><i>level</i></code></pre>
+  <div class="alert"> Deprecated in release GEE 5.2.5 and higher. Use <code>--maxleveloverride</code></a> instead.</div>
+  <p><i>Optional</i>. Sets the maximum level for the imagery. Uses internal Fusion scale. Deprecated because it not match the levels specified in the Fusion UI.</p>
+  <pre>
+  <code><b>--maxleveloverride </b><i>level</i></code></pre>
+  <p><i>Optional</i>. Sets the maximum level for the imagery. Uses the imagery scale in Fusion. Matches the levels in the Fusion UI.</p>
+
+  <h2>gemodifyterrainproject</h2>
+  <pre>
+ <code><b>gemodifyterrainproject</b> [--<strong>mercator</strong> | --<strong>flat]</strong>
+  -o <em>projectname</em> {--<strong>maxlevel</strong> <em>level</em> | --<strong>maxleveloverride</strong> <em>level</em>]}
+  [--<strong>no_terrain_overlay</strong> | {--<strong>terrain_overlay</strong> --<strong>start_level</strong> <em>level</em> --<strong>resource_min_level</strong> <em>level</em> }]
+  <em>insetresource</em>} </code>...</pre>
+ 
+
+  <h3>Purpose</h3>
+  <p>Modifies resources of an existing terrain project. </p>
```

Comment:
I think `Modifies resources of an existing terrain project.` should be changed to `Modifies an existing terrain project.` similar to the wording of `gemodifyimageryproject`.
This command can modify the project settings in addition to the resources.

Reply:
Modified.